examples/basic/index.ts

```diff
@@ -0,0 +1,90 @@
+import { AppLoaderPlugin, Shadow } from 'pixi-shadows';
+import { Application, Container, InteractionEvent, SCALE_MODES, Sprite, Texture } from 'pixi.js';
```

Comment:
```suggestion
import { Application, Container, InteractionEvent, SCALE_MODES, Sprite, Texture } from 'pixi.js';
import bgTexturePath from '../../assets/background.jpg';
import demonTexturePath from '../../assets/flameDemon.png';
```

It seems that we need to import the used textures to make sure vite outputs them to the `dist` directory and link them correctly. Additionally if you do this, we need to add some global TS declarations to the examples directory so TS remains happy:
```ts
declare module '*.jpg' {
    const data: string;

    export default data;
}

declare module '*.png' {
    const data: string;

    export default data;
}
```

The same of course would apply in all the other examples if we take this route.

Reply:
yes! good catch
instead of custom global typings I'm adding `"types": ["vite/client"]` to `tsconfig.json` and it seems to do the trick.